sdk/template/template/samples/javascript/helloWorld.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+async function main() {
+  console.log("== Hello World ==");
+  // TODO
```

Comment:
I think we can call the client here.

Reply:
I am going to completely rework the samples in this package within the next week anyway 😄 , so deferring this to later.

Reply:
Ended up doing it anyway because CI gets upset if no samples call the client.